source/_patterns/02-molecules/card/_card.scss

```diff
@@ -1,3 +1,5 @@
+@import '../../00-base/05-colors/_colors.scss';
```

Comment:
A note here that our new Namespaces task should generate `@base, @atoms, @molecules, @organisms` for both Twig and Webpack.